docs/content/en/integrations/parsers/file/bearer_cli.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Bearer CLI"
+toc_hide: true
+---
+
+### File Types
+DefectDojo parser accepts a .json file.
+
+To export a .json file from Bearer CLI, pass "-f json" to your Bearer command  
+See Bearer documentation: https://docs.bearer.com/reference/commands/
+
+### Sample Scan Data
+Sample Bearer scans can be found [here](https://github.com/DefectDojo/django-DefectDojo/tree/master/unittests/scans/bearer)
+
+### Acceptable JSON Format
+
+~~~
+
+{
+    "critical": [
```

Comment:
I don't know if this is necessary.  You use this as a duplicate within the parser.py. Remove either one.

Reply:
Did so as other scanner do this as well. Documentation is easier to find/read as source code for some as others find the odd way.

Reply:
I agree with manual-sommer here.

We've settled on linking to the files used for unit testing the parser instead of dropping a copy of the file format in the documentation.

For an example, see this:
https://github.com/DefectDojo/django-DefectDojo/blob/master/docs/content/en/integrations/parsers/file/checkmarx.md

which renders like this in the documentation site:
https://documentation.defectdojo.com/integrations/parsers/file/checkmarx/